ibrcommon/configure.ac

```diff
@@ -51,6 +51,10 @@ AC_SUBST(GENERIC_LIBRARY_VERSION)
 AC_SUBST(GENERIC_LIBRARY_NAME)
 AC_SUBST(GENERIC_VERSION)
 
+dnl Drops autotools defaults but keeps user-specific options.
+: ${CFLAGS=""}
```

Comment:
I have never seen this syntax. What is the colon for? Is cross-compiling using buildchains as OpenWrt, buildroot, or openembedded still possible or will all given CFLAGS be cleared?

Reply:
This is from [autoconf manual](https://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/C-Compiler.html), about AC_PROG_CC.

Custom C[XX]FLAGS are honored, e.g.
`CXXFLAGS="-Dsomething" ./configure && make` produces

g++ -DHAVE_CONFIG_H -I../.. -O2 -Dsomething -MT vaddress.lo -MD -MP -MF .deps/vaddress.Tpo -c -o vaddress.lo vaddress.cpp

Is this enough for cross-build?

Reply:
Yes, that should work. Thanks for the source reference.